codec-http/src/main/java/io/netty/handler/codec/http/DefaultFullHttpResponse.java

```diff
@@ -35,36 +37,92 @@ public class DefaultFullHttpResponse extends DefaultHttpResponse implements Full
      */
     private int hash;
 
+    /**
+     * Create an empty HTTP response with the given HTTP version and status.
+     */
     public DefaultFullHttpResponse(HttpVersion version, HttpResponseStatus status) {
-        this(version, status, Unpooled.buffer(0));
+        this(version, status, Unpooled.buffer(0), DEFAULT_HEADER_FACTORY, DEFAULT_TRAILER_FACTORY);
```

Comment:
The above comments will also help to make internal usage consistent, bcz right now `DefaultFullHttpResponse` and `DefaultFullHttpRequest` use different references to the same default builder